facebook-commerce.php

```diff
@@ -1339,6 +1341,35 @@ public function update_product_group( WC_Facebook_Product $woo_product ) {
 		}
 	}
 
+	/**
+	 * Creates a product item using the facebook Catalog Batch API. This replaces existing functionality,
+	 * which is currently using facebook Product Item API implemented by `WC_Facebookcommerce_Integration::create_product_item`
+	 *
+	 * @since x.x.x
+	 * @param WC_Facebook_Product $woo_product
+	 * @param string              $retailer_id
+	 **/
+	public function create_product_item_batch_api( $woo_product, $retailer_id, $product_group_id ): string {
+		try {
+			$product_data        = $woo_product->prepare_product( $retailer_id, \WC_Facebook_Product::PRODUCT_PREP_TYPE_ITEMS_BATCH );
+			$requests            = WC_Facebookcommerce_Utils::prepare_product_requests_items_batch( $product_data );
+			$facebook_catalog_id = $this->get_product_catalog_id();
+			$response            = $this->facebook_for_woocommerce->get_api()->send_item_updates( $facebook_catalog_id, $requests );
+
+			if ( $response->handles ) {
```

Comment:
I noticed that in other places, we do this. We can keep this if we are sure about the `handles` property in the response:

``` php
if ( isset( $response_handles ) && is_array( $response_handles ) )